mdsuite/calculators/radial_distribution_function.py

```diff
@@ -59,7 +62,7 @@ class RadialDistributionFunction(Calculator, ABC):
     loaded_property : str
             Property loaded from the database_path for the analysis
     minibatch: int, default None
-            Size of a individual minibatch, if set. By default minibatching is not applied
+            Size of a individual minibatch, if set. By default mini-batching is not applied
 
     See Also
     --------
```

Comment:
the example below should be changed to RDF from ADF

Reply:
Yep.